package.json

```diff
@@ -7,13 +7,36 @@
     "web": "expo start --web",
     "eject": "expo eject",
     "prepare": "husky install",
-    "lintfix": "eslint --fix --ext .js,.jsx ."
+    "lintfix": "eslint --fix --ext .js,.jsx .",
+    "test": "jest",
+    "testDebug": "jest -o --coverage=false",
+    "testNoCov": "jest --coverage=false",
+    "updateSnapshots": "jest -u --coverage=false",
+    "type": "module"
```

Comment:
not sure if we actually need this

Reply:
why?

Reply:
the tests wouldn't run initially and I thought this might help, I just removed it again